dev/MRTCore/mrt/Core/src/MRM.cpp

```diff
@@ -993,7 +993,9 @@ STDAPI MrmGetFilePathFromName(_In_opt_ PCWSTR filename, _Outptr_ PWSTR* filePath
         if (SUCCEEDED(wil::TryGetEnvironmentVariableW(L"MICROSOFT_WINDOWSAPPRUNTIME_BASE_DIRECTORY", baseDir)) && baseDir)
         {
             path.swap(baseDir);
-            RETURN_IF_FAILED(StringCchLengthW(path.get(), STRSAFE_MAX_CCH, &bufferCount));
+            RETURN_IF_FAILED(StringCchLengthW(path.get(), STRSAFE_MAX_CCH, &length));
+            size_t bufferCount;
+            RETURN_IF_FAILED(SizeTAdd(length, 1, &bufferCount));
```

Comment:
>bufferCount [](http://example.com/codeflow?start=50&length=11)

Adding to this throwaway inner variable?

Reply:
fixed